perfil/election/management/commands/__init__.py

```diff
@@ -0,0 +1,21 @@
+from perfil.election.models import Election
+from perfil.party.models import Party
+from perfil.person.models import Person
+
+
+def election_keys(election):
```

Comment:
initally I thought about putting this in the ImportCSVCommand or another class to be inherited by our commands, so you wouldn't have to import it everytime. Don't have any problem to leave it this way, but maybe you didn't think about the other possibility

Reply:
It cannot be a method of `ImportCsvCommand` (or `Command`) otherwise we cannot access in in the definition of our custom `Command`s. What I mean is that where we have:

```python
class Command(ImportCsvCommand):

    model = …
    bulk_size = …
    to_cache (Election, election_keys)
    …
```

It would be an invalid Python code to use:

```python
class Command(ImportCsvCommand):

    model = …
    bulk_size = …
    to_cache (Election, self.election_keys)
    …
```

This is invalid because`self` doesn't exist yet.  Some alternatives I've though about (but discarded them all):

* using `(Election, ImportCsvCommand.election_key)` would be possible if we use the `@staticmethod` decorator, but unnecessarily verbose IMHO
* Defining it as the default method to cache `Election` so we don't have to pass it in the `to_cache` instance variable would work, but would put some boundaries in the way we do lookups in the database and cache for each given model (making it difficult or unnecessarily verbose to customize)
* We could define it per `load_whatever.py` file, but we would repeat code in our code base (`election_keys` as it is is used in two or three commands I guess)

That's why it is the way it is. Does it make sense? Do you prefer any other implementation? Let me know. IMHO importing it every time is tolerable comparing to the downside of the three alternatives I could think of. What about you?

Reply:
It makes total sense. Thank you for the detailed explanation :)